workflows/chipseq.nf

```diff
@@ -296,12 +309,33 @@ workflow CHIPSEQ {
     //
     // MODULE: Picard post alignment QC
     //
+    BAM_FILTER_BAMTOOLS
+                .out
+                .bam
+                .map {
+                    [ it[0], it[1], [] ]
+                }
```

Comment:
What is this for?

Reply:
A `view` that I forgot to remove, thanks for spotting it!

Reply:
Fix in [6a75945](https://github.com/nf-core/chipseq/pull/339/commits/6a75945dc4bacacb93f11941a09eeec8539f2d39)